### PR TITLE
Configured subscription integration

### PR DESCRIPTION
**Pull request by jeremie6wind**
Hello,
This is the first part of the configured subscription integration.
Enable the configured feature in setup.sh.
Add the new yang files to setup.sh.
Add the yang files to the project with TLS disabled.
Direct definition of the receiver in the subscription is disabled. Only reference receiver instances is enabled.
This avoids this warning:

root@ubuntu2204:~/sshfs_root# netopeer2-cli 
> connect -u
ly WARNING: Schema node "transport" for parent "/ietf-subscribed-notifications:subscriptions/subscription" not found; in expr "derived-from(../../transport" with context node "/ietf-subscribed-notifications:subscriptions/subscription/receivers/receiver".

### CODE REVIEW

File: modules/ietf-udp-client.yang [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  import ietf-inet-types {
+    prefix inet;
+    reference
+      "RFC 6991: Common YANG Data Types";
+  }
+  /*import ietf-tls-client {
+    prefix tlsc;
+    reference
+      "RFC TTTT: YANG Groupings for TLS Clients and TLS Servers";
+  }*/
```
Comment (jktjkt) [inline modules/ietf-udp-client.yang:15]: Please put this into a separate commit with a commit message that explains why it needs to be done. Also, please add these YANG models in an extra commit which clearly states where they were taken from, which RFC/draft it is.

**Review by michalvasko — COMMENTED** [review]
I have created a separate branch for your changes. Mentioning the modules' draft revision/revision date is not a bad idea, though.

**Comment by jeremie6wind** [comment]
Thank you for creating the udp-notif branch. 
Do you expect anything else in the commits/commit logs?

**Review by michalvasko — APPROVED** [review]

**Comment by michalvasko** [comment]
Sorry, must have missed the changes.
